tests/phpunit/unit/Extension/TwigTraitTest.php

```diff
@@ -11,7 +13,45 @@
  */
 class TwigTraitTest extends BoltUnitTest
 {
-    public function test()
+    public function testTwigExtension()
     {
+        $app = $this->getApp(false);
+        $ext = new TwigExtension();
+        $baseDir = $app['filesystem']->getDir('extensions://');
+        $baseDir->setPath('local/bolt/twig');
+        $ext->setBaseDirectory($baseDir);
+        $ext->setContainer($app);
+        $ext->register($app);
+        $app->boot();
+
+        $this->addToAssertionCount(1);
+    }
+
+    public function testRenderTemplate()
+    {
+        $app = $this->getApp();
+        $template = <<< TWIG
+Function koala {{ koala(name) }}
+Function dropbear {{ dropbear(name) }}
+Filter koala {{ name|koala }}
+Filter dropbear {{ name|dropbear }}
+TWIG;
+        $loader = new ArrayLoader(['marsupial.twig' => $template]);
+        $app['twig']->setLoader($loader);
```

Comment:
Yeah, but I CBF iterating test to perfection … they're tests, better things to focus on.